modules/S115_usermode_emulator.sh

```diff
@@ -81,13 +83,24 @@ S115_usermode_emulator() {
   fi
 }
 
+filesystem_fixes() {
+  if [[ "${#MISSING[@]}" -ne 0 ]]; then
+    sub_module_title "Filesystem fixes"
+    print_output "[*] Emba has auto-generated the files during runtime."
+    print_output "[*] For persistence you could generate it manually in your filesystem.\\n"
+    for MISSING_FILE in "${MISSING[@]}"; do
+      print_output "[*] Missing file: $MISSING_FILE"
```

Comment:
Something went wrong here: 
```
[*] Missing file: /etc/resolv.conf
[*] Missing file: /etc/TZ
[*] Missing file: 

==> Software component and version detection started
```

Reply:
Additional: Maybe at the end one example command line to create all these files: `touch x1 x2 x3 ...` for copying?

Reply:
This is what emba is doing but if you would do something more usefull you neet to do it in some other way ;)
So, the "touch something" example does not help.

Reply:
I thought it could help, but that only was an idea :)